en/docs/get-started/tutorials/create-your-first-rest-api.md

```diff
@@ -27,273 +26,60 @@ In this section, let's develop the application that retrieves COVID-19 related s
     | **Name**        | `COVID-19Stats`                       |
     | **Description** | `Get statistics relating to COVID-19` |
 
-4. Click **Create**.
-
-    This takes you to the **Components** page shown below.
-
-    ![Create component](../../assets/img/tutorials/rest-api/create-component.png){.cInlineImage-full}
+4. Click **Create**. This takes you to the **Components** page.
 
 ### Step 1.2: Add a REST API component
 
 Let's create a new REST API component as follows:
 
-1. On the **Components** page, click **Create**.
+1. On the **Components** page, click **+Create** on the the **REST API** card.
 
-2. Click the **REST API** card.
+    ![Create component](../../assets/img/tutorials/rest-api/create-component.png){.cInlineImage-small}
 
-3. Click **Create an API from scratch**.
-
-4. Enter a unique name and a description for the API. For this tutorial, let's enter the following values:
+2. Enter a unique name and a description for the API. For this tutorial, let's enter the following values:
 
     | **Field**       | **Value**             |
     |-----------------|-----------------------|
     | **Name**        | `Statistics`          |
     | **Description** | `COVID-19 Statistics` |
 
 5. Click **Next**.
+6. Provide authorization for Choreo to connect to your GitHub account by clicking  **Authorize with GitHub**. 
+7. If you have not already authorized Choreo applications, click **Authorize Choreo Apps** when prompted.
+8. Select a GitHub account, a repository that includes the implementation, the relevant branch, and the build preset: Ballerina or Dockerfile. 
```

Comment:
Here also the user has to select the Ballerina preset.

Reply:
same as above,

Reply:
Since the steps after this only instruct users to enter a path, we are assuming that the user has selected **Ballerina** (which may not necessarily be the case). Therefore, let's  include the sentence "For this tutorial, let's select **Ballerina**."

Reply:
We are not assuming the user has selected Ballerina, I am just keeping it generic as possible until we get the implementation related details sorted out. Lets send this as a generic detail for the first cut.

Reply:
If we are keeping it generic, then the steps that follow this step will have to fork out to cater to either option. This is because if the user who follows this guide clicks **Docker** at this point, he/she will see a few other fields like Dockerfile, Context, etc. that are relevant to Docker instead of the **Path** field.